tests/test_winkernel.cpp

```diff
@@ -0,0 +1,149 @@
+/* Second-Best Disassembler Engine */
+/* By Nguyen Anh Quynh <aquynh@gmail.com>, 2013-2016 */
```

Comment:
Changed names in some files with 882052a. 

And understood about the new branch. Thanks for creating a new branch.